ChefExtensionHandler/bin/chef-uninstall.sh

```diff
@@ -16,7 +16,7 @@ uninstall_ubuntu_chef_package(){
 
```

Comment:
@NimishaS The above code is really necessary now ? Since, now we are using apt-get here?